src/main/java/io/micronaut/build/AbstractDockerMojo.java

```diff
@@ -42,7 +42,7 @@
      * </a> configuration options.
      */
     @Parameter(property = "micronaut.native-image.args")
-    protected String nativeImageBuildArgs;
+    protected List<String> buildArgs;
```

Comment:
Should this have been renamed? I suppose it only relates to native build?

Reply:
Yes, only to native build. I named it the same as the Native Image Maven Plugin, as you suggested.

However, I'm realizing that this is a breaking change :/